src/db_ref.ts

```diff
@@ -111,17 +112,26 @@ export class DBRef extends BSONValue {
     return new DBRef(doc.$ref, doc.$id, doc.$db, copy);
   }
 
-  /** @internal */
-  [Symbol.for('nodejs.util.inspect.custom')](): string {
-    return this.inspect();
-  }
+  inspect(depth?: number, options?: unknown, inspect?: InspectParameterFn): string {
+    const addQuotes = inspect ? false : true;
+    inspect ??= basicInspectParameterFn;
+
+    const namespaceArg = addQuotes
+      ? `'${inspect(this.namespace, options)}'`
+      : inspect(this.namespace, options);
+    const objectIDArg = addQuotes
+      ? `new ObjectId('${inspect(this.oid, options)}')`
+      : inspect(this.oid, options);
+    const args = [namespaceArg, objectIDArg];
+
+    if (this.db) {
+      args.push(addQuotes ? `'${inspect(this.db, options)}'` : inspect(this.db, options));
+    }
+
+    if (Object.keys(this.fields).length > 0) {
+      args.push(inspect(this.fields, options));
+    }
 
```

Comment:
```suggestion
    const args = [
     inspect(this.namespace, options),
     inspect(this.oid, options),
     
     // this is stylistic - but a commonly used technique to conditionally add items to an array using the `...` operator
     ...(this.db ? [inspect(this.db, options)] : []),
     ...(Object.keys(this.fields).length > 0 ? inspect(this.fields, options) : [])
    ].map(arg => addQuotes ? `'${arg}'` : arg);
    
    return `new DBRef(${args.join(', ')})`;
```

optional.  specifically, I like the idea of building up the args array and then conditionally wrapping all the args in quotes instead of using a ternary operator for each argument individually.  how we build up the array is another optional suggestion - I like using the `...` operator with the ternary operator but that's personal preference.

Reply:
Sounds good, I committed these changes and added one small tweak to args[1].